llvm/lib/Target/AMDGPU/InstCombineTables.td

```diff
@@ -1,5 +1,8 @@
 include "llvm/TableGen/SearchableTable.td"
-include "llvm/IR/Intrinsics.td"
+include "llvm/Target/Target.td"
+
+// Dummy target to pacify SearchTableEmitter code.
+def DummyTarget : Target;
```

Comment:
It was better before, this is adding more work for another tablegen invocation

Reply:
Agreed, though most other invocations seem conformant, i.e., use the Target.td instead of Intrinsics.td directly. And it does create less plumbing. But I am find going back to the earlier impl if you insist.

Reply:
If we're going to require associating the intrinsics with a target, it would be better to actually include the AMDGPU target definition instead of defining a DummyTarget

Reply:
AMDGPU.td